test/scf_compare.jl

```diff
@@ -52,6 +65,41 @@ include("testcases.jl")
     end
 end
 
+@testset "Compare different SCF algorithms (collinear spin, no temperature)" begin
```

Comment:
Strange use case, but ok ...

Reply:
Well, it's just to test this spin thing for safety, but temperature is not implemented for newton yet. I agree that it has no real interest...